terraform/modules/emr/templates/emr/setup.sh

```diff
@@ -42,34 +42,34 @@ for GROUP in $${COGNITO_GROUPS[@]}; do
   sudo groupadd -f "$GROUP"
 
   echo "Adding users for group $GROUP"
-  USERS=$(aws cognito-idp list-users-in-group --user-pool-id "${user_pool_id}" --group-name "$GROUP" | jq '.Users' | jq -r '.[].Username')
+  USERS=$(aws cognito-idp list-users-in-group --user-pool-id "${user_pool_id}" --group-name "$GROUP" | jq '.Users[]' | jq -r '(.Attributes[] | if .Name =="preferred_username" then .Value else empty end) // .[].Username')
```

Comment:
This doesn't run. Need to remove the '.[].' from the end (so it reads // .Username)

Reply:
good catch - I had removed that, but somehow made it back in